test/packages/schema.js

```diff
@@ -0,0 +1,59 @@
+import {isUrl} from '../validators.js';
+import {iso31662} from 'iso-3166';
+
+export default {
+  name: {
+    type: String,
+    required: true,
+    length: {
+      min: 2,
+      max: 200,
+    },
+  },
+  title: {
+    en: {
+      type: String,
+      required: true,
+      length: {
+        min: 4,
+        max: 200,
+      },
+    },
+    fr: {
+      type: String,
+      required: true,
+      length: {
+        min: 4,
+        max: 200,
+      },
+    },
+  },
+  jurisdiction: {
+    type: String,
+    required: true,
+    use: {isIso31662},
+  },
+  source: {
```

Comment:
Let's unify this `source` with the `showcase` entries and list both `repository` and `license`.